modules/wordads/wordads.php

```diff
@@ -491,6 +477,40 @@ function get_ad_snippet( $section_id, $height, $width, $location = '', $css = ''
 HTML;
 	}
 
+	/**
+	 * Returns the completed ad div with snipped to be inserted into the page
+	 *
+	 * @param  string  $spot top, side, inline, or belowpost
+	 * @param  string  $snippet The snippet to insert into the div
+	 * @param  array  $css_classes
+	 * @return string The supporting ad unit div
+	 *
+	 * @since 7.1
+	 */
+	function get_ad_div( $spot, $snippet, array $css_classes = array() ) {
+		if ( empty( $css_classes ) ) {
+			$css_classes = array();
+		}
+
+		$css_classes[] = 'wpcnt';
+		if ( 'top' == $spot ) {
+			$css_classes[] = 'wpcnt-header';
+		}
+
+		$classes = esc_attr( implode( ' ', $css_classes ) );
+		$about  = __( 'Advertisements', 'jetpack' );
```

Comment:
I don't think it would hurt if we had 

```suggestion
		$about  = esc_html__( 'Advertisements', 'jetpack' );
```